development/jobguard/README.md

```diff
@@ -11,24 +11,30 @@ The main purpose of Job Guard is to delay running integration jobs that depend o
 
 To run the application, use this command:
 
```

Comment:
After removing the command you forgot about this line. 
Please move the command from the **Example** section here, and remove the `### Run the application` line above. 
See also the READMEs for [prowjobparser](https://github.com/kyma-project/test-infra/tree/cef5cedd57404c2aa971662f9bff926ad3b90285/development/prowjobparser) and [image-syncer](https://github.com/kyma-project/test-infra/tree/cef5cedd57404c2aa971662f9bff926ad3b90285/development/image-syncer). 
``````suggestion
To run the application, use this command:

```shell
go run cmd/jobguard/main.go \
  -github-endpoint=http://ghproxy \
  -github-endpoint=https://api.github.com \
  -github-token-path=/path/to/oauth \
  -org=example-org \
  -repo=example-repo \
  -base-ref=13abc \
  -expected-contexts-regexp="(some-context-regexp|another-context)"
```
``````